configs/offload-server.yaml

```diff
@@ -15,7 +15,7 @@ plugin_groups:
       - name: OpenVINS
         path:
            git_repo: https://github.com/ILLIXR/open_vins.git
-           version: "3.1"
```

Comment:
Let's merge this in OpenVINS to 3.2 before accepting this PR.

Reply:
Yeah sure. Please have the [OpenVINS PR](https://github.com/ILLIXR/open_vins/pull/15) and the [Kimera-VIO PR](https://github.com/ILLIXR/Kimera-VIO/pull/9) reviewed together with this one.